### PR TITLE
feat: allow all css possibilities for default template

### PR DESCRIPTION
**Pull request by rishabh3112**
**What kind of change does this PR introduce?**
feature

**Did you add tests for your changes?**
Done

**If relevant, did you update the documentation?**
No

**Summary**
Allow all possible css solutions to the users

**Does this PR introduce a breaking change?**
No

**Other information**
N/A

### CODE REVIEW

**Comment by rishabh3112** [comment]
Tests are failing because of yarn registry issues. Works fine locally.

**Comment by rishabh3112** [comment]
Again 😞

**Comment by rishabh3112** [comment]
CI is green ✅ 
/cc @webpack/cli-team need review. Need to do further work dependent on this one.

**Review by alexander-akait — COMMENTED** [review]
Ready for review? I think make sense to merge firstly https://github.com/webpack/webpack-cli/pull/2547#pullrequestreview-619592164 and rewrite some tests on shapshots

File: test/init/__snapshots__/init.test.js.snap.webpack4 [hunk 1]
```diff
@@ -457,7 +512,59 @@ module.exports = {
                 test: /\\\\.s[ac]ss$/i,
                 use: ['style-loader', 'css-loader', 'sass-loader'],
             },
+            {
+                test: /\\\\.(eot|svg|ttf|woff|woff2|png|jpg|gif)$/,
```
Comment (alexander-akait) [inline test/init/__snapshots__/init.test.js.snap.webpack4:516]: Let's use `i` flag for regexp, but let's do it in the next PR

**Review by alexander-akait — APPROVED** [review]
/cc @webpack/cli-team

**Review by snitin315 — APPROVED** [review]

**Comment by alexander-akait** [comment]
cache problem again, should be fixed after https://github.com/webpack/webpack-cli/pull/2553

**Review by anshumanv — APPROVED** [review]

File: test/init/__snapshots__/init.test.js.snap.webpack4 (hunk 1 above)
Comment (rishabh3112) [reply test/init/__snapshots__/init.test.js.snap.webpack4:516]: Okay, on it!

**Pull request merged**